packages/ui/components/form/timezone-select/TimezoneSelect.tsx

```diff
@@ -33,20 +33,29 @@ export function TimezoneSelect({
     });
   }, [components]);
 
+  // We use modifiedTimezones in place of the allTimezones object replacing any underscores in the curly braces
+  // with spaces and removing the America/Detroit timezone, adding the America/New_York timezone instead.
+  const modifiedTimezones = useMemo(() => {
+    const { "America/Detroit": _, ...rest } = allTimezones;
```

Comment:
I think a better approach would be to fork react-timezone-select and change it there. It's a simple  https://github.com/ndom91/react-timezone-select/blob/main/src/timezone-list.ts#L15 file. Also, maybe worth opening a PR there if they want to incorporate this. cc @zomars 

If someone changes Detroit to something else in the package, it would stop working. So, it isn't a stable change to do. Good as a hack only.

Reply:
Is worth generate a [patch-package](https://github.com/ds300/patch-package) for this?

Reply:
that's a good idea! I'm not a fan of forking npm packages

Reply:
@zomars Wow!! Never heard about patch-package. This is what we need here.